infra/token-transfer-client/src/constants/index.js

```diff
@@ -13,7 +13,7 @@ if (process.env.NODE_ENV === 'production') {
     apiUrl = process.env.INVESTOR_API_URL
   }
 } else {
-  apiUrl = 'http://localhost:5000'
+  apiUrl = 'http://103.228.153.6:5000'
```

Comment:
This must be your IP? Let's keep localhost since most developers will be running the stack on their local environment. But we could have an override to make it easier to run the API remotely. Something like:
```
apiUrl = process.env.DEV_API_URL || 'http://localhost:5000'
```

Reply:
@franckc  Sorry, My bad. I will update this

Reply:
@franckc I have changed this

Reply:
no problem! thanks for fixing it.